.buildkite/pipelines/security_solution_quality_gate/mki_periodic/mki_periodic_investigations.yml

```diff
@@ -17,3 +17,37 @@ steps:
       automatic:
         - exit_status: '-1'
           limit: 1
+
+  - group: "API MKI - Investigations"
+    key: api_test_investigations
+    steps:
+      - label: Running investigations:saved-objects:runner:qa:serverless
+        command: .buildkite/scripts/pipelines/security_solution_quality_gate/api_integration/api-integration-tests.sh investigations:saved-objects:runner:qa:serverless
+        key: investigations:saved-objects:runner:qa:serverless
+        agents:
+          image: family/kibana-ubuntu-2004
+          imageProject: elastic-images-prod
+          provider: gcp
+          machineType: n2-standard-4
+          preemptible: true
+        timeout_in_minutes: 120
+        retry:
+          automatic:
+            - exit_status: "1"
+              limit: 2
+
+    steps:
```

Comment:
steps here introduce a duplicate key. I dont know if it will be working. Remove steps line as it is mentioned above.